sessions/recap-project-5/art-gallery-app/README.md

```diff
@@ -169,6 +167,32 @@ To achieve this, we’ll divide the tasks into two parts:
 
 ### 5.1. Favorite Button
 
+Users want the ability to add additional information to each art piece. Consider where to store the art piece data to ensure all pages can access it efficiently. Remember that we’ll be reusing this favorite button across multiple pages.
+
+<details>
+<summary>Hint ✨</summary>
+
+In Next.js, the recommended place to store data or states that need to be accessed across the app is in `_app.js`.
+
+It’s best practice to consolidate related information into a single state. However, if you're not comfortable with this approach, you can use separate states. Just ensure you can clearly identify which information corresponds to each art piece—consider saving a unique identifier, such as a slug, within each state for clarity.
+
+**Example:**
+
+- **Using Multiple States:**
+
+   ```js
+   const [carColors, setCarColors] = useState({ model: "vw-taos", colors: ["red", "blue", "white"] });
+   const [carAutomatic, setCarAutomatic] = useState({ model: "vw-taos", automatic: true });
+   ```
```

Comment:
Ah well I think this version does not have the benefits of the multi state solution. Please checkout my comment.

Reply:
I combine them.